lang/cs/Org.Apache.REEF.Client/Common/DriverFolderPreparationHelper.cs

```diff
@@ -154,8 +154,18 @@ internal void CreateDefaultFolderStructure(AppParameters appParameters, string d
             File.WriteAllText(Path.Combine(driverFolderPath, _fileNames.GetBridgeExeConfigPath()), config);
 
             // generate .config file for Evaluator executable
-            File.WriteAllText(Path.Combine(driverFolderPath, _fileNames.GetGlobalFolderPath(), EvaluatorExecutable), 
-                DefaultDriverConfigurationFileContents);
+            var userDefinedEvaluatorConfigFile = Path.Combine(JarFolder, EvaluatorExecutable);
+            var evaluatorConfigFile = Path.Combine(driverFolderPath, _fileNames.GetGlobalFolderPath(), EvaluatorExecutable);
+            if (!File.Exists(userDefinedEvaluatorConfigFile))
+            {
+                 File.WriteAllText(evaluatorConfigFile, DefaultDriverConfigurationFileContents);
+                 Logger.Log(Level.Info, "Create EvaluatorConfigFile with default config");
```

Comment:
It would be helpful to add the name of the file written, as it would make it easier to find this in the logs.

Reply:
Done